File: codeforces contests/div4/Codeforces Round 928 (Div. 4)/b.cpp
#include <iostream>
#include <vector>
#include <string>
using namespace std;

int main() {
    int t;
    cin >> t;
    
    for (int i = 0; i < t; i++) {
        int n;
        cin >> n;
        vector<int> count;
        for (int j = 0; i < n; i++)
        {
            int c = 0;
            string s;
            cin >> s;
            c.count('1')
            if (c==0) {
                continue;
            }
            count.append(c);
        }
        if (max(count)==min(count)) {
            cout<<"SQUARE"
        } else {
            cout<<"TRIANGLE"
        }
        
    }

File: codeforces contests/div4/Codeforces Round 928 (Div. 4)/b.py
t = int(input())
for i in range(t):
    n = int(input())
    count = []
    
    for j in range(n):
        c=0
        s=input()
        c=s.count("1")
        if c==0:
            continue
        count.append(c)

    if max(count)==min(count):
        print("SQUARE")
    else:
        print("TRIANGLE")